charts/telemetry-collector/values.yaml

```
@@ -0,0 +1,167 @@
+global: {}
+
+nameOverride: ""
+fullnameOverride: ""
+
+replicaCount: 1
+
+bootstrap:
+  create: true
+  name: ""
+  kafka:
+    enabled: true
+    hostname: ""
+    port: 0
+  nats:
+    enabled: true
+    hostname: ""
+    port: 0
+
+image:
+  repository: accelleran/telemetry-collector
+  pullPolicy: IfNotPresent
+  tag: "0.9.0"
```

Leave this empty to use the `appVersion` by default.

```suggestion
  tag: ""
```

---

OK